crates/sui-core/src/consensus_adapter.rs

```diff
@@ -66,7 +66,9 @@ const SEQUENCING_CERTIFICATE_LATENCY_SEC_BUCKETS: &[f64] = &[
     20., 25., 30., 60.,
 ];
 
-const SEQUENCING_CERTIFICATE_POSITION_BUCKETS: &[f64] = &[0., 1., 2., 3., 5., 10.];
+const SEQUENCING_CERTIFICATE_POSITION_BUCKETS: &[f64] = &[
+    0., 1., 2., 3., 5., 10., 15., 20., 25., 30., 50., 100., 150., 200.,
+];
```

Comment:
Nice! Since you are doing this maybe increase the buckers for the `SEQUENCING_CERTIFICATE_LATENCY_SEC_BUCKETS` as well? Maybe additionally `90, 120, 180, 250, 300`